server/models/user-schema.js

```diff
@@ -20,33 +20,18 @@ const UserSchema = new mongoose.Schema(
     },
     following: {
       type: [String]
+    },
```

Comment:
From what I see here, the user schema holds the data for pictures that they post. However, would the user then not require a "dankness score" value added to the model as well, since each picture is rated? I ask this because I notice that the picture description and comment sections were already prepared.

Reply:
A dankness score will have to be added, but I was unsure if each post would get a unique dankness score or if there would be an overall dankness score for the profile. Either way, that addition can be made after the like/dislike feature has been developed, as that is the basis for its functionality.